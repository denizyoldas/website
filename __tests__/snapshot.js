import { render } from '@testing-library/react'
import Index from '../src/pages/index'
import About from '../src/pages/about'
import Projects from '../src/pages/projects'

it('renders homepage unchanged', () => {
  const { container } = render(<Index />)
  expect(container).toMatchSnapshot()
})

it('renders about unchanged', () => {
  const { container } = render(<About />)
  expect(container).toMatchSnapshot()
})

it('renders projects unchanged', () => {
  const { container } = render(<Projects />)
  expect(container).toMatchSnapshot()
})
