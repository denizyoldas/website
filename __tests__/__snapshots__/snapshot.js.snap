// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders about unchanged 1`] = `
<div>
  <h3
    class="underline underline-offset-4 pt-3"
  >
    Hakkımda
  </h3>
  <p>
    Meslek lisesinden mezun olmadan staj yaparak yazılım geliştirmeye başladım. Bu süreçte Full-Stack olarak görev aldım. Sonrasında üniversiteye gidip aynı dönemde çalışma hayatında bulunmaya devam ettim. Genel olarak Start-Up firmalarında yer aldım. Üniversiteden sonra kariyer hayatıma Front-End Developer olarak devam ediyorum.
  </p>
  <h4
    class="underline underline-offset-3"
  >
    Deneyim
  </h4>
  <div
    class="border-b-2 border-slate-500"
  >
    <h3
      class="font-bold mb-0"
    >
      Front-End Developer
    </h3>
    <h6
      class="font-extralight"
    >
      Dijital Kurye · Ekim 2021 - Mevcut
    </h6>
    <ul>
      <li>
        <p>
          Dijital kurye adı verilen projede Angular 11 kullanarak bir operasyon yönetimi, imza yönetimi vb... sistemlerin yönetildiği bir dashboard geliştirmekteyim. RxJs, Fabric, PdfJS gibi önemli kütüphaneler kullanarak birçok farklı geliştirme yaptım.
        </p>
      </li>
      <li>
        <p>
          "Canlı Müzayede" adı verilen uygulamada ise react kullanarak geliştirme yapıyorum. Burada Redux, Material, CraftJs gibi teknolojilere yer veriyorum.
        </p>
      </li>
      <li>
        <p>
          Electron.js kullanarak ekran-ses-ekran sesi olacak şekilde kayıt uygulaması geliştirdim.
        </p>
      </li>
      <li>
        <p>
          "Meksa verifier" adı verilen MVC projesinde JQuery ile geliştirmeler yaptım.
        </p>
      </li>
    </ul>
  </div>
  <div
    class="border-b-2 border-slate-500"
  >
    <h3
      class="font-bold mb-0"
    >
      Front-End Developer
    </h3>
    <h6
      class="font-extralight"
    >
      Altamira Digital · Ekim 2020 - Temmuz 2021
    </h6>
    <ul>
      <li>
        <p>
          Angular 8 kullanarak geliştirdiğim projelerin başında "CondLife" adı verilen yönetim paneli platformu geliyor. Avm, Site, Rezidans yönetimi için yapılan bir dashboard paneli. Diğer projelerde olduğu gibi RxJs, WebSocket vb. kütüphaneler kullanarak geliştirmeler yaptım.
        </p>
      </li>
      <li>
        <p>
          "Borusan Kios" Projesi geliştirdim. Angular 8 ve Electron.js kullanarak Borusan Otomotiv için tamamiyle SignalR ile çalışan bir Para Ödeme Kiosk'u geliştirmesinde yer aldım.
        </p>
      </li>
      <li>
        <p>
          Istanbul'da bulunan "Acarkent" sitesinin yönetimi için tasarlanmış dashboard'a Angular 8 ile geliştirmelerde bulundum.
        </p>
      </li>
    </ul>
  </div>
  <div
    class="border-b-2 border-slate-500"
  >
    <h3
      class="font-bold mb-0"
    >
      Full-Stack Developer
    </h3>
    <h6
      class="font-extralight"
    >
      İdeal Bilişim · Eylül 2018 - Kasım 2019
    </h6>
    <ul>
      <li>
        <p>
          PHP kullanarak mobil uygulama için API yazdım.
        </p>
      </li>
      <li>
        <p>
          Laravel Framework'ü kullanarak Veteriner yönetim uygulaması geliştirdim.
        </p>
      </li>
      <li>
        <p>
          HTML, CSS, JQuery kullanarak websiteleri geliştirmesi yaptım.
        </p>
      </li>
      <li>
        <p>
          Ionic - Angular 2 kullanarak mobil uygulama geliştirdim.
        </p>
      </li>
      <li>
        <p>
          Angular.js kullanarak Front-End geliştirmeleri yaptım.
        </p>
      </li>
    </ul>
  </div>
  <h4
    class="underline underline-offset-3 pt-3"
  >
    Yetenekler
  </h4>
  <div
    class="grid grid-cols-2 gap-4 sm:grid-cols-4 text-center"
  >
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      Angular
    </span>
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      Angular.js
    </span>
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      React
    </span>
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      React Native
    </span>
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      Vue
    </span>
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      JQuery
    </span>
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      Electron
    </span>
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      RxJs
    </span>
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      HTML
    </span>
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      CSS
    </span>
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      JavaScript
    </span>
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      TypeScript
    </span>
    <span
      class="bg-slate-500 rounded-lg text-white"
    >
      Go
    </span>
  </div>
</div>
`;

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="pt-8"
  >
    <div
      class="text-6xl font-medium"
    >
      Deniz Aksu
    </div>
    <div
      class="text-sm"
    >
      Frontend Developer
       
      <span
        class="text-xs"
      >
        (React, Next.js, TypeScript, Angular, Node.js)
      </span>
    </div>
    <div
      class="text-center bg-gray-200 dark:bg-gray-600 rounded-md mt-4"
    >
      Eğlenceli bir yazılım geliştiricisi
    </div>
    <h3
      class="underline underline-offset-4"
    >
      Kısaca
    </h3>
    <p>
      4 yılı aşkın bir süredir yazılım geliştirmekte olup kendinimi geliştirmeye ve ileriye taşımaya odaklanan bir yazılımcıyım. Lise yıllarında
       
      <span
        class="text-purple-600 dark:text-yellow-500"
      >
        javascript
      </span>
       
      ile tanışmamdan sonra genel olarak
       
      <span
        class="text-purple-600 dark:text-yellow-500"
      >
        javascript
      </span>
       
      temelli geliştirmeler yaptım.
    </p>
    <h3
      class="underline underline-offset-4 pt-3"
    >
      Benimle iletişime geçmek için
    </h3>
    <p>
      Geliştirdiğim projeleri takip etmek isterseniz github'a, attığım tweetlere bakmak için twitter hesabıma bakabilirsiniz. Detaylı iletişim için mail ile ulaşım sağlayabilirsiniz.
    </p>
    <div
      class="flex place-content-center"
    >
      <div
        class="flex flex-row justify-evenly w-1/3"
      >
        <a
          aria-label="github"
          class="hover:underline"
          href="https://github.com/denizyoldas"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            class="hover:text-yellow-300"
            fill="currentColor"
            height="24"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.012 8.012 0 0 0 16 8c0-4.42-3.58-8-8-8z"
            />
          </svg>
        </a>
        <a
          area-label="twitter"
          class="hover:underline"
          href="https://twitter.com/yoldasaksu"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            class="hover:text-blue-300"
            fill="currentColor"
            height="24"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M5.026 15c6.038 0 9.341-5.003 9.341-9.334 0-.14 0-.282-.006-.422A6.685 6.685 0 0 0 16 3.542a6.658 6.658 0 0 1-1.889.518 3.301 3.301 0 0 0 1.447-1.817 6.533 6.533 0 0 1-2.087.793A3.286 3.286 0 0 0 7.875 6.03a9.325 9.325 0 0 1-6.767-3.429 3.289 3.289 0 0 0 1.018 4.382A3.323 3.323 0 0 1 .64 6.575v.045a3.288 3.288 0 0 0 2.632 3.218 3.203 3.203 0 0 1-.865.115 3.23 3.23 0 0 1-.614-.057 3.283 3.283 0 0 0 3.067 2.277A6.588 6.588 0 0 1 .78 13.58a6.32 6.32 0 0 1-.78-.045A9.344 9.344 0 0 0 5.026 15z"
            />
          </svg>
        </a>
        <a
          area-label="linkedin"
          class="hover:underline"
          href="https://www.linkedin.com/in/denizyoldas/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            class="hover:text-blue-400"
            fill="currentColor"
            height="24"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z"
            />
          </svg>
        </a>
        <a
          area-label="email"
          class="hover:underline"
          href="mailto:daksu30@gmail.com?subject=Web Sitesi Üzerinden"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            class="hover:text-yellow-300"
            fill="currentColor"
            height="24"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 1024 1024"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M477.5 536.3L135.9 270.7l-27.5-21.4 27.6 21.5V792h752V270.8L546.2 536.3a55.99 55.99 0 0 1-68.7 0z"
            />
            <path
              d="M876.3 198.8l39.3 50.5-27.6 21.5 27.7-21.5-39.3-50.5z"
            />
            <path
              d="M928 160H96c-17.7 0-32 14.3-32 32v640c0 17.7 14.3 32 32 32h832c17.7 0 32-14.3 32-32V192c0-17.7-14.3-32-32-32zm-94.5 72.1L512 482 190.5 232.1h643zm54.5 38.7V792H136V270.8l-27.6-21.5 27.5 21.4 341.6 265.6a55.99 55.99 0 0 0 68.7 0L888 270.8l27.6-21.5-39.3-50.5h.1l39.3 50.5-27.7 21.5z"
            />
          </svg>
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders projects unchanged 1`] = `
<div>
  <h3
    class="underline underline-offset-4 pt-3"
  >
    Projeler
  </h3>
  <ul
    class="grid grid-flow-row gap-10 place-items-center"
  >
    <li
      class="group flex flex-col items-center drop-shadow-lg text-center py-4 px-6 bg-white dark:bg-neutral-900 max-w-md"
    >
      <a
        href="https://wordonce.com/"
        rel="nopper noreferrer"
        target="_blank"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27400%27%20height=%27300%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="Wordonce Game"
            class="w-full h-auto rounded-md object-contain"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
      </a>
      <div
        class="flex flex-col items-center"
      >
        <h3
          class="text-2xl font-bold"
        >
          Wordonce Game
        </h3>
        <span>
          20-00-2022
        </span>
        <p
          class="text-sm"
        >
          Vue kullanarak freelance olarak geliştirdiğim bir kelime oyunu. Kısaca satırda veya sütünda aynı harf bulunmaması ve okunduğunda anlamlı bir cümle oluşması kurallarına dayanan bir oyun.
        </p>
        <div
          class="grid grid-cols-2 gap-3"
        >
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            Vue
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            TypeScript
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            Pinia
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            Vite
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            sass
          </span>
        </div>
      </div>
    </li>
    <li
      class="group flex flex-col items-center drop-shadow-lg text-center py-4 px-6 bg-white dark:bg-neutral-900 max-w-md"
    >
      <a
        href="https://maiorganizasyon.com/"
        rel="nopper noreferrer"
        target="_blank"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27400%27%20height=%27300%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="Mai Organizasyon"
            class="w-full h-auto rounded-md object-contain"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
      </a>
      <div
        class="flex flex-col items-center"
      >
        <h3
          class="text-2xl font-bold"
        >
          Mai Organizasyon
        </h3>
        <span>
          04-00-2022
        </span>
        <p
          class="text-sm"
        >
          Mai Organizasyon web sitesi tasarımı, kodlanması ve google işletmem hesabı kurulumu.
        </p>
        <div
          class="grid grid-cols-2 gap-3"
        >
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            React
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            Next.js
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            TypeScript
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            sass
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            tailwindcss
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            Figma
          </span>
        </div>
      </div>
    </li>
    <li
      class="group flex flex-col items-center drop-shadow-lg text-center py-4 px-6 bg-white dark:bg-neutral-900 max-w-md"
    >
      <a
        href="https://google.com/chrome-bookmarks-extension"
        rel="nopper noreferrer"
        target="_blank"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27400%27%20height=%27300%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="Google Chrome Bookmarks Extension"
            class="w-full h-auto rounded-md object-contain"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
      </a>
      <div
        class="flex flex-col items-center"
      >
        <h3
          class="text-2xl font-bold"
        >
          Google Chrome Bookmarks Extension
        </h3>
        <span>
          27-00-2022
        </span>
        <p
          class="text-sm"
        >
          Google Chrome Bookmarks Extension
        </p>
        <div
          class="grid grid-cols-2 gap-3"
        >
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            React
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            TypeScript
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            sass
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            Styled Components
          </span>
        </div>
      </div>
    </li>
    <li
      class="group flex flex-col items-center drop-shadow-lg text-center py-4 px-6 bg-white dark:bg-neutral-900 max-w-md"
    >
      <a
        href="https://portfolio-denizyoldas.vercel.app"
        rel="nopper noreferrer"
        target="_blank"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27400%27%20height=%27300%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="Portfolio"
            class="w-full h-auto rounded-md object-contain"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
      </a>
      <div
        class="flex flex-col items-center"
      >
        <h3
          class="text-2xl font-bold"
        >
          Portfolio
        </h3>
        <span>
          27-00-2021
        </span>
        <p
          class="text-sm"
        >
          React js kullanarak youtube da gördüklerime ekleyerek oluşturdum eski portfolio sitem.
        </p>
        <div
          class="grid grid-cols-2 gap-3"
        >
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            React
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            sass
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            css
          </span>
        </div>
      </div>
    </li>
    <li
      class="group flex flex-col items-center drop-shadow-lg text-center py-4 px-6 bg-white dark:bg-neutral-900 max-w-md"
    >
      <a
        href="https://github.com/denizyoldas/hotel-app"
        rel="nopper noreferrer"
        target="_blank"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27400%27%20height=%27300%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="Hotel App"
            class="w-full h-auto rounded-md object-contain"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
      </a>
      <div
        class="flex flex-col items-center"
      >
        <h3
          class="text-2xl font-bold"
        >
          Hotel App
        </h3>
        <span>
          20-00-2022
        </span>
        <p
          class="text-sm"
        >
          Bir case olarak gelen ve yaparken keyif aldığım. Kendim için geliştirdiğim bir otel puanlama ekleme uygulaması
        </p>
        <div
          class="grid grid-cols-2 gap-3"
        >
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            React
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            TypeScript
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            Tailwindcss
          </span>
        </div>
      </div>
    </li>
    <li
      class="group flex flex-col items-center drop-shadow-lg text-center py-4 px-6 bg-white dark:bg-neutral-900 max-w-md"
    >
      <a
        href="#"
        rel="nopper noreferrer"
        target="_blank"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27400%27%20height=%27300%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="Screen Record App"
            class="w-full h-auto rounded-md object-contain"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
      </a>
      <div
        class="flex flex-col items-center"
      >
        <h3
          class="text-2xl font-bold"
        >
          Screen Record App
        </h3>
        <span>
          24-00-2022
        </span>
        <p
          class="text-sm"
        >
          Electron js kullanarak geliştirdiğim ekran kayıt uygulaması. Ekranın sesini ve bilgisayarın mikrofonundan gelen sesi kayıt altına alıp belirlenen dosyaya mp4 uzantılı bir çıktı alarak izlemenize olanak sağlıyor.
        </p>
        <div
          class="grid grid-cols-2 gap-3"
        >
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            Electron
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            Bulma
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            JavaScript
          </span>
        </div>
      </div>
    </li>
    <li
      class="group flex flex-col items-center drop-shadow-lg text-center py-4 px-6 bg-white dark:bg-neutral-900 max-w-md"
    >
      <a
        href="https://github.com/denizyoldas/todo-app"
        rel="nopper noreferrer"
        target="_blank"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27400%27%20height=%27300%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="ToDo App"
            class="w-full h-auto rounded-md object-contain"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
      </a>
      <div
        class="flex flex-col items-center"
      >
        <h3
          class="text-2xl font-bold"
        >
          ToDo App
        </h3>
        <span>
          13-00-2022
        </span>
        <p
          class="text-sm"
        >
          React Native kullanarak geliştirdiğim bir not tutma uygulaması. Genel olarak ios uyumlu bir app olarak basit düzeyde not tutmaya yardımcı bir uygulama
        </p>
        <div
          class="grid grid-cols-2 gap-3"
        >
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            React Native
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            Chakra UI
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            TypeScript
          </span>
          <span
            class="text-xs dark:bg-gray-700 dark:text-white bg-gray-200 rounded-md py-1 px-2"
          >
            Jotai
          </span>
        </div>
      </div>
    </li>
  </ul>
</div>
`;
